deploy/openshift/deploy-kiali-to-openshift.sh

```diff
@@ -229,3 +229,23 @@ do
     exit 1
   fi
 done
+
+# Deploy Kiali MonitoringDashboards to OpenShift
+# Note for undeploy script: dashboards are implicitely undeployed when the related CRD is removed
+echo "Deploying Kiali dashboards to OpenShift project ${NAMESPACE}"
+for dashboard in nodejs thorntail vertx-client vertx-eventbus vertx-pool vertx-server
+do
+  yaml_path="${YAML_DIR}/../dashboards/${dashboard}.yaml"
+  if [ -f "${yaml_path}" ]; then
+    echo "Using YAML file: ${yaml_path}"
+    cat ${yaml_path} | envsubst | ${OC_TOOL_PATH} apply -n ${NAMESPACE} -f -
+  else
+    get_downloader
+    yaml_url="https://raw.githubusercontent.com/kiali/kiali/${VERSION_LABEL}/deploy/dashboards/${dashboard}.yaml"
+    echo "Downloading YAML via: ${downloader} ${yaml_url}"
+    ${downloader} ${yaml_url} | envsubst | ${OC_TOOL_PATH} apply -n ${NAMESPACE} -f -
+  fi
+  if [ "$?" != "0" ]; then
+    echo "WARNING: Failed to deploy runtimes dashboards. If you need them, please deploy manually."
```

Comment:
same comments as above - "implicitly" and see what other way we can word this WARNING message to make it clear what failed and that it doesn't break base Kiali functionality.